src/pages/console/uibuilder/figmatocode.mdx

```diff
@@ -62,6 +61,11 @@ import { ComponentA, ComponentB }  from './ui-components';
 
 ### Figma file changelog
 
+#### 1.2.0: July 25 2022
+
+* Add TextAreaField primitive component. For additional information on converting existing TextFields to TextAreaFields, see [here](/console/uibuilder/textfieldtoarea/).
+* Update supported Amplify UI React version to 3.1.0.
+
```

Comment:
Maybe out of scope here, but are we missing changelogs for 1.1.0?

Reply:
Added a backlog task to update this changelog